_posts/0000-01-02-shubhamkothawade24.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
created a branch, 
created a file 
and made a commit,
 and opened a
 pull request